tests/Dockerfile

```diff
@@ -0,0 +1,27 @@
+ARG MAVEN_VERSION=3-amazoncorretto-8
```

Comment:
I picked this particular tag because Nextflow's docker image is based on [amazoncorretto](https://github.com/nextflow-io/nextflow/blob/e952299f74aa2d209e408fe91b8dd783f04b64f3/docker/Dockerfile#L1) - there's no further logic than that.